tests/SprykerSdkTest/Spryk/Console/SprykDumpTest.php

```diff
@@ -40,6 +40,33 @@ public function testDumpsAllSpryks()
         $tester->execute($arguments);
 
         $output = $tester->getDisplay();
-        static::assertRegExp('/List of all Spryk definitions/', $output);
+        $this->assertRegExp('/List of all Spryk definitions/', $output);
+    }
+
+    /**
+     * @return void
+     */
+    public function testDumpsSpecficSpryk()
```

Comment:
You have a typo here `testDumpsSpecficSpryk` should be `testDumpsSpecificSpryk `

Reply:
Fixed.